api/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "api",
   "version": "1.0.0",
-  "main": "index.js",
+  "main": "server.js",
```

Comment:
if you are going to change the main file make sure to make this change everywhere that it is used for example 

```json
"start": "node index.js",
 "dev": "node --watch index.js",
 "dev:debug:inspect": "node --inspect --watch index.js"
```